cpp/src/transform/jit/operation-udf.hpp

```diff
@@ -1,8 +1,5 @@
 /*
- * Copyright (c) 2019-2020, NVIDIA CORPORATION.
- *
- * Copyright 2018-2019 BlazingDB, Inc.
- *     Copyright 2018 Christian Noboa Mardini <christian@blazingdb.com>
```

Comment:
I don't know if we get to delete this...

Reply:
git diff is messing up, `code.h` was deleted in its entirety. `operation-udf.hpp` is a new file, effectively blank.

Reply:
Blazing copyright is maintained in the new files where appropriate, I think.